examples/Results/main.roc

```diff
@@ -0,0 +1,106 @@
+app [main] {
+    pf: platform "https://github.com/roc-lang/basic-cli/releases/download/0.12.0/Lb8EgiejTUzbggO2HVVuPJFkwvvsfW6LojkLR20kTVE.tar.br",
+}
+
+import pf.Stdout
+import pf.Task exposing [Task]
+
+# This function parses a string formatted like "{FirstName} {LastName} was born in {Year}"
+# This is the most verbose version. It's a bit too nested, we will do better below.
+parsePersonVerbose = \line ->
+    when line |> Str.split " was born in " is
```

Comment:
Can we change all the use of `Str.split` to `Str.splitFirst`? Want to make sure we recommend the best api here. It still return s a result, so should be good for showing err handling

Reply:
Also should mean we can remove `toPair` below, which I think will help cleanup the example.

It will just be something like:
{before: firstName, after: bithYearStr} <- Str.splitFirst line " was born in " |> Result.try

Reply:
Great recommendations, thanks. This is fixed now.